src/components/sync-connected-sites.tsx

```diff
@@ -132,10 +132,14 @@ const SyncConnectedSitesSection = ( {
 
 	const mainSite = section.connectedSites.find( ( item ) => ! item.isStaging );
 	const hasConnectionErrors = mainSite?.syncSupport !== 'already-connected';
-	const sitePullState = getPullState( selectedSite.id, section.id );
-	const isPulling = sitePullState && isKeyPulling( sitePullState.status.key );
-	const sitePushState = getPushState( selectedSite.id, section.id );
-	const isPushing = sitePushState.isInProgress;
+	const isAnySectionSitePulling = section.connectedSites.some( ( site ) => {
```

Comment:
I think `isPulling` still makes sense as a simpler name.

Reply:
Thanks for the feedback. I have updated the names in https://github.com/Automattic/studio/pull/706/commits/2d9451d3c5ab018b2879cfb455f7fd2fec7c00a1 and merged the PR.